src/main/java/controller/UserAPI.java

```diff
@@ -59,20 +64,20 @@
 
         String path = request.getRequestURI();
         requestValidator.signupRequestValidator(data, path);
-        /* necessary fields */
+        /** necessary fields */
```

Comment:
This is not javadoc, but an implementation comment.
You should use `// ... ` or `/* ... */` style for implementation comments.

For small implementation comments, like this one, prefer using the single line variant - `// ...`

Reply:
Done.